dev-python/pyphen/pyphen-0.9.4-r1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python{2_7,3_6,3_7,3_8} )
+DISTUTILS_USE_SETUPTOOLS=bdepend
+PLOCALES="af af_ZA bg bg_BG ca cs cs_CZ da da_DK de de_AT de_CH de_DE el el_GR en en_GB en_Latn_GB en_Latn_US en_US es et et_EE fr gl hr hr_HR hu hu_HU is it it_IT lt lt_LT lv lv_LV nb nb_NO nl nl_NL nn nn_NO pl pl_PL pt pt_BR pt_Latn_BR pt_Latn_PT pt_PT ro ro_RO ru ru_RU sk sk_SK sl sl_SI sr sr_Latn sv te te_IN uk uk_UA zu zu_ZA"
+
+inherit distutils-r1 l10n
```

Comment:
drop l10n

Reply:
Could you elaborate on that please? I use the l10n for the l10n_find_plocales_changes function to see if there are changed dictionaries between versions.